fbw-a380x/src/base/flybywire-aircraft-a380-842/de-DE.locPak

```diff
@@ -7,8 +7,8 @@
       "AIRCRAFT.UI_MODEL": "A320neo",
       "AIRCRAFT.UI_VARIATION": "Airbus",
 
-      "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
-      "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Ausgewählten Steuerkurs nach rechts anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
+      "A380X.COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
```

Comment:
you shouldn't even have these files in the base folder any more!?

Reply:
> Those came after a merge conflict.... If there's still a file diff in here it's that file is still on master isn't it? Should we remove those from master first maybe?

they are not on master any more - I removed them there - see screenshot

https://github.com/flybywiresim/aircraft/tree/master/fbw-a380x/src/base/flybywire-aircraft-a380-842
![image](https://github.com/flybywiresim/aircraft/assets/16833201/8e2e25c9-e4a9-4b92-8c42-606c2ce1c55b)